.changeset/little-jars-train.md

```diff
@@ -0,0 +1,7 @@
+---
+"miniflare": minor
+---
+
+Fix JOIN behaviour when selecting columns with the same name.
```

Comment:
can we pls use the recommended [changesets format](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changeset-message-format) 🙏